### DISCUSSION TITLE
Adding New Integrations to the Authelia Guide

### DISCUSSION THREAD

**Discussion by Forged-111a** [authelia/authelia#8223] [Documentation Feedback] [answered]
Hello, I've been using Authelia for over a year now, and I really like the project. However, the integration documentation has some gaps. For example, I recently integrated Authelia with GlitchTip (a Sentry alternative) via OIDC, and I'd like to share my configuration with the community. How can I do this?

**Comment by james-d-elliott** [comment, marked as answer]
Naturally we can't document any app, and all app integrations are community contributions which we're very grateful for. It's not so much a gap as it is documenting every application that exists is just something we can't possibly achieve even with help from the community, let alone by ourselves.

You can either post your examples here, using `auth.example.com` and `glitchtip.example.com` as domain examples and the same secret every other example app has, including any neatly cropped pictures, or do the same and open a [pull request](https://github.com/authelia/authelia/pulls). Ideally you'd use an existing similar app as an example and build off of that, for example [Portainer](https://github.com/authelia/authelia/tree/master/docs/content/integration/openid-connect/portainer) (for a fairly standard Auth Code flow app with a GUI), [ArgoCD](https://github.com/authelia/authelia/tree/master/docs/content/integration/openid-connect/argocd) (for an app with a web app and CLI app using purely config files), [GitLab](https://github.com/authelia/authelia/tree/master/docs/content/integration/openid-connect/gitlab) (for a fairly standard Auth Code web app), or [ExpressJS](https://github.com/authelia/authelia/tree/master/docs/content/integration/openid-connect/expressjs) (for a develop your own app style app).

**Reply by james-d-elliott** [reply]
@Forged-111a just added a few examples, most of which were curated by me but have been updated by others.

**Reply by hendrik1120** [reply]
GlitchTip also uses the django-allauth library for oidc like some other apps in the integration guide do. However, GlichTips docs recommend the gui approach through django admin. Let me know if you have any questions or issues.
Django-allauth 65.2.0 will also add the ability to enable PKCE using that gui, but it will likely take a release cycle for it to be added to GlitchTip.

**Reply by Forged-111a** [reply]
I’d like to specifically describe the configuration through the graphical interface because, if set up incorrectly, the SSO authorization button doesn’t appear in the authorization window. This can create the misleading impression that OIDC isn’t working.

Honestly, this particular issue is what pushed me to write a guide, as I spent several days searching for the problem in the wrong place. I haven't encountered this issue with any other setups where I’ve configured Authelia.

**Reply by hendrik1120** [reply]
Interesting, looking forward to reading your guide. I will take a look into their implementation.
